comps/llms/summarization/tgi/llm.py

```diff
@@ -7,7 +7,7 @@
 from langchain.chains.summarize import load_summarize_chain
 from langchain.docstore.document import Document
 from langchain.text_splitter import CharacterTextSplitter
-from langchain_community.llms import HuggingFaceEndpoint
+from langchain_huggingface import HuggingFaceEndpoint
```

Comment:
By doing this change, HuggingfaceEndpoint will not check ${HuggingfaceHub_API_TOKEN} env variables